resources/address_format/CH.json

```diff
@@ -7,7 +7,9 @@
 		"locality",
 		"postalCode"
 	],
-	"uppercase_fields": [],
```

Comment:
This is odd, how did that happen? Difference in json encoding between our systems?